eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -133,6 +133,10 @@ jobs:
         Windows_NetFramework:
           OSVmImage: "windows-2019"
           TestTargetFramework: net461
+        Windows_NetFramework_ProjectReferences:
```

Comment:
@weshaggard FYI.

We are adding .NET Framework specific code that's on a very hot path adding a configuration to make sure it's tested before the release.

Reply:
Do you intend for this to stay longer term? If it doesn't add too much cost I'm fine with it staying but I know we are going to start testing a larger matrix soon and will likely look for candidates to trim on default builds.

Reply:
We would be able to remove it after new features ship as a package.